packages/nuka/src/hooks/use-measurement.test.tsx

```diff
@@ -150,8 +150,36 @@ describe('useMeasurement', () => {
 
     const { totalPages, scrollOffset } = result.current;
 
-    expect(totalPages).toBe(3);
-    expect(scrollOffset).toEqual([0, 573, 1146]);
+    expect(totalPages).toBe(4);
+    // 573 * 0, 573 * 1, 573 * 2, 573 * 3 + (1720 - 573 * 3)
```

Comment:
Not sure if this is useful, but I thought a code comment explaining where the values are coming from for the `toEqual` might be helpful?
Happy to edit or delete if it adds more confusion for now